### PR TITLE
Maxey/platform

### PR DESCRIPTION
**Pull request by mikemaxey**
### Description
Please describe your pull request.

💔Thank you!

### CODE REVIEW

File: platform/authorities.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Authorities
+html_title: Smallstep Platform | Authorities, powered by step-ca
+description: "/docs/platform - Learn about the Smallstep Platform"
```
Comment (alanchrt) [inline platform/authorities.mdx]: Descriptions should be a description of the page, specific to the content, and not include the URL. It appears under the title in a Google search result.

File: platform/index.mdx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: Smallstep Platform
+html_title: Smallstep Platform
+description: "/docs/platform - Learn about the Smallstep Platform"
+unfurl: /static/graphics/smallstep-docs-unfurl.png
+---
+
+# Platform
```
Comment (alanchrt) [inline platform/index.mdx]: Should this be _The Smallstep Platform_?

File: platform/provisioners.mdx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+- **Cloud API Provisioners** - Useful for issuing certificates to public cloud virtual machines, 
+Cloud API Provisioners use the native cloud provider API and instance identity documents to automate certificates. 
+With support for AWS, GCP, and Azure metadata APIs, the Cloud API provisioner accelerates secure cloud operations.
+- **JWK Provisioner** - Useful for a broad range of workflows, the JWK provisioner provides a flexible JSON Web Token-based authentication flow. 
+Often paired with infrastructure automation solutions, the JWK Provisioner can deliver one-time tokens to a new workload to later be exchanged for an x.509 certificate.
+
+## Learn More
+
+Provisioners are an open-source feature. 
+Visit the `[step-ca` documentation](https://smallstep.com/docs/step-ca/certificate-authority-core-concepts#provisioners) for a complete list of Provisioners and instructions on usage and capabilities of each.
```
Comment (alanchrt) [inline platform/provisioners.mdx]: Looks like the backtick outside the `[` is breaking this link.

File: platform/templates.mdx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+Templates allow you to add custom SANs or extensions:
+
+- Customize certificates for a broad range of applications and integrations.
+- Provides full control of all of certificate fields, even add custom extensions.
+- Automate custom certificate generation and skip the manual workflows.
+
+## Learn More
+
+Templates are an open-source feature. 
+Visit the `[step-ca` documentation](https://smallstep.com/docs/step-ca/configuration#certificate-templates) for a complete description of Templates and instructions on usage and capabilities.
```
Comment (alanchrt) [inline platform/templates.mdx]: Same with this one.. the misplaced backtick breaks the link.

**Comment by tashian** [comment]
I'm getting this error when I try to load the docs pages in this branch locally:

Screen Shot 2021-03-31 at 12 39 46 PM

File: platform/index.mdx (hunk 2 above)
Comment (mikemaxey) [reply platform/index.mdx]: yes

File: platform/authorities.mdx (hunk 1 above)
Comment (mikemaxey) [reply platform/authorities.mdx]: added to each page

File: platform/provisioners.mdx (hunk 3 above)
Comment (mikemaxey) [reply platform/provisioners.mdx]: fixed

File: platform/templates.mdx (hunk 4 above)
Comment (mikemaxey) [reply platform/templates.mdx]: fixed

**Pull request merged**